baremetal/metal3machine_manager.go

```diff
@@ -1323,6 +1323,7 @@ func (m *MachineManager) GetProviderIDAndBMHID() (string, *string) {
 	// instead contains / to separate the names. In that case we return nil for
 	// the bmh ID to force the controller to fetch it differently.
 	if strings.Contains(bmhID, "/") {
+		m.Log.Info("ProviderID is in new format, it does not contain the BMH ID", "providerID", providerID)
 		return *providerID, nil
 	}
 	return *providerID, pointer.StringPtr(bmhID)
```

Comment:
Should not add also logs here as suggested in the issue

Reply:
@zhouhao3 Sorry If was not clear I meant we keep the log in line 1326 and add another similair one after the line 1328 saying it does contain the bmhID

Reply:
Updated, thanks.